modules/post/windows/manage/run_as.rb

```diff
@@ -121,5 +121,7 @@ def run
       vprint_status("Thread Id: #{pi[:thread_id]}")
       print_status("Command output:\r\n#{tmpout}") unless tmpout.nil?
     end
+    print_status("Cleaning up...")
+    rm_f(outpath)
```

Comment:
Arguably, you shouldn't even be writing the file if `CMDOUT` isn't set. That should be fixed.